src/index.js

```diff
@@ -4,7 +4,7 @@ import { version } from '../package.json';
 lazyload.version = version;
 lazyload.install = function (Vue) {
     // eslint-disable-next-line no-undef
-    Vue.directive(Slider.name, Slider);
+    Vue.component(Slider.name, Slider);
```

Comment:
From where do you get this `Slider`, still I don't know... Could you explain to me this part of your code?

Reply:
OMG!!!

Reply:
there should be a lazyload

Reply:
pls change it

Reply:
Thanks :)